src/main/java/io/citrine/jcc/search/dataset/result/DatasetMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.dataset.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a dataset multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class DatasetMultiSearchResultElement extends BaseMultiSearchResultElement<DatasetSearchResult> {
```

Comment:
A WILD LURKER EMERGES.

I agree that this could be kind of nice for end users (of which there are 2 in the world 😄 ), but I'm a little worried about the precedent it sets. @kjaym if I contribute to the codebase do you expect this pattern to be followed? It seems like significantly more work than it's worth. This pattern makes sense a lot more sense in scala (despite the fact that type inference means you don't have to write out the type unless you want to anyway).